apps/new-dealer/views/partials/important-message.html

```diff
@@ -1,4 +1,7 @@
-<div class="notice">
-  <span class="icon-important"></span>
-  <strong class="important-text">{{#t}}pages.{{step}}.important{{/t}}</strong>
-</div>
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    {{#t}}pages.{{step}}.important{{/t}}
+  </strong>
+</div>
```

Comment:
New line?

Reply:
I have now removed this file as it is no longer needed - warning messages have been built into hof.